### ISSUE TITLE
buchhaltung commit should work with hledger

### ISSUE THREAD

**Issue by varac** [johannesgerer/buchhaltung#12]
I'm using hledger and started using buchhaltung - first of all thank you for this project !!!
It would be great if `buchhaltung commit` would would with hledger, currently it doesn't:

```
--- ~/hledger ‹master* AM⁇› » buchhaltung commit -m'First steps with buchhaltung match'
buchhaltung: ledger: readCreateProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
```

**Comment by johannesgerer** [comment]
It should not require many changes. Any suggestion on the user interface? introduce another commit subcommand for hledger like `hci` ?

**Comment by varac** [comment]
What about a setting in the config file which tells buchhaltung whether ledger or hledger is used, and bucchaltung figures out by itself how to deal with it ?

**Comment by johannesgerer** [comment]
Sounds resonable. Until someone implements that, a workaround would be to create an alias for hledger, i.e.
````
sudo ln -s /usr/bin/hledger /usr/bin/
````

**Comment by johannesgerer** [comment]
Fixed by 65ab169, which allows use the commands `commitHledger` or `ch` for short.
